src/DataTable/TableCell.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TableCell = ({ cell }) => (
+  <td {...cell.getCellProps()}>
+    <span className="pgn__table-cell-wrap">
+      {cell.render('Cell')}
+    </span>
+  </td>
+);
+
+TableCell.propTypes = {
+  cell: PropTypes.shape({
```

Comment:
Same feedback as `TableRow` have you considering making the cell object itself the props for `TableCell` (`<TableCell {...cell} />`)